dslink-scala-nest/src/main/scala/com/uralian/nest/service/NestHttpService.scala

```diff
@@ -92,4 +103,26 @@ class NestHttpService(client: NestHttpClient) extends NestService {
     client.httpGet[T](s"devices/thermostats/$deviceId/$name").recover {
       case NonFatal(e) => throw new IllegalArgumentException(s"Error retrieving devices/thermostats/$deviceId/$name", e)
     }
+
+  /**
+    * Read a stream of data and send it to sink
+    *
+    * @param sink
+    * @param token
+    * @param ec
+    * @param system
+    * @tparam T
+    * @return
+    */
+  def readThermostatStream[T](sink: StreamSink[T])(implicit token: AccessToken, ec: ExecutionContext,
```

Comment:
Ok, just thinking loud here: what are benefits or passing a sink vs not passing anything and simply return `Source[T, _]`?